snippets/python/config/config.py

```diff
@@ -0,0 +1,158 @@
+import requests
+
+content_type = "application/vnd.netbackup+json; version=4.0"
+
+
+def perform_login(username, password, base_url, domain_name, domain_type):
+    url = base_url + "/login"
+
+    if domain_name != "" and domain_type != "":
+        req_body = {"userName": username, "password": password, "domainName": domain_name, "domainType": domain_type}
+    else:
+        req_body = {"userName": username, "password": password}
+
+    headers = {'Content-Type': content_type}
+
+    print("performing POST on {} for user '{}'\n".format(url, req_body['userName']))
+
+    resp = requests.post(url, headers=headers, json=req_body, verify=False)
+
+    if resp.status_code != 201:
+        raise Exception('Login API failed with status code {} and {}'.format(resp.status_code, resp.json()))
+
+    return resp.json()['token']
+
+def get_media_servers(jwt, base_url):
+    url = base_url + "/config/media-servers"
+    headers = {'Content-Type': content_type, 'Authorization': jwt}
+    query_params = {
+        #	"page[limit]": 100, 				#This changes the default page size to 100
+        #	"filter": "jobType eq 'RESTORE'"	#This adds a filter to only show RESTORE Jobs
```

Comment:
Again, the purpose of the commented-out parameters is unclear. Either remove or explain, please.